src/Taggable/TaggablePSR6PoolAdapter.php

```diff
@@ -261,13 +264,6 @@ public function invalidateTags(array $tags)
         // Remove all items with the tag
         $success = $this->deleteItems($itemIds);
 
-        if ($success) {
-            // Remove the tag list
-            foreach ($tags as $tag) {
-                $this->removeList($this->getTagKey($tag));
```

Comment:
We cannot remove this. The TaggablePSR6PoolAdapter must make sure to manage all the tags itself. It cannot trust the actual `$this->cachePool`